roles/schema_registry/tasks/main.yml

```diff
@@ -156,6 +156,21 @@
     - rbac_enabled|bool
     - ((not oauth_enabled) or ( oauth_enabled and ldap_with_oauth_enabled ))
 
+- name: Import IDP certificate to Truststore for OAuth
+  include_role:
+    name: common
+    tasks_from: idp_certs.yml
+  vars:
+    idp_cert_path: "{{ oauth_idp_cert_path }}"
+    idp_cert_dest: "oauth_idp_cert.pem"
+    alias: "oauth_cert"
+    truststore_storepass: "{{schema_registry_truststore_storepass}}"
+    truststore_path: "{{schema_registry_truststore_path}}"
+    create_bouncy_castle_keystore: false
+  when:
+    - oauth_enabled|bool or schema_registry_oauth_enabled|bool or kafka_rest_oauth_enabled|bool or kafka_connect_oauth_enabled|bool or kafka_connect_replicator_oauth_enabled|bool
+    - oauth_idp_cert_path != ""
```

Comment:
since the varaible `oauth_idp_cert_path != ""` is same as `idp_self_signed == true` should we `idp_self_signed` that here for more clarity while reading this and all other components

Reply:
idp_self_signed is an internal variable for us, which is why I have not added any comment with that variable in defaults. We can change it, it essentially means the same thing but it's better to not highlight it and confuse the customers. They don't need to use this var

Reply:
I didnt mean we should add comment for this internal variable but rather I meant here in this task and similar ones where the condition is `oauth_idp_cert_path != ""` we can change it to `idp_self_signed | bool` for making it slightly easier to read maybe. Not a big deal just a small thought.

Reply:
I mean the reason I did not use the variable is because the customers go through our playbooks and we don't want to highlight an internal variable when both these variables do the same job, I agree it's cleaner with the bool var which is why I used it for creating the configs and filters

Reply:
fair enough